aspnet/conceptual-overview/aspnet.rst

```diff
@@ -1,37 +1,37 @@
+ASP.NET Core is a new open-source and cross-platform framework for building modern cloud-based Web apps using .NET. We built it from the ground up to provide an optimized development framework for apps that are deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core apps cross-platform on Windows, Mac and Linux. ASP.NET Core is open source at `GitHub <https://github.com/aspnet/home>`_.
 
 Why build ASP.NET Core?
 -----------------------
 
-The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.  Since then millions of developers have used it to build and run great web applications, and over the years we have added and evolved many, many capabilities to it.
+The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.  Since then millions of developers have used it to build and run great web apps, and over the years we have added and evolved many capabilities to it.
 
-With ASP.NET Core we are making a number of architectural changes that make the core web framework much leaner and more modular. ASP.NET Core is no longer based on System.Web.dll, but is instead based on a set of granular and well factored NuGet packages allowing you to optimize your app to have just what you need. You can reduce the surface area of your application to improve security, reduce your servicing burden and also to improve performance in a true pay-for-what-you-use model.
+ASP.NET Core has a number of architectural changes that result in a much leaner and modular framework. ASP.NET Core is no longer based on *System.Web.dll*, but is instead based on a set of granular and well factored `NuGet <http://www.nuget.org/>`__ packages allowing you to optimize your app to include just the NuGet packages you need. You can reduce your apps surface area to improve security, reduce  servicing and improve performance in a pay-for-what-you-use model.
 
-ASP.NET Core is built with the needs of modern Web applications in mind, including a unified story for building Web UI and Web APIs that integrate with today's modern client-side frameworks and development workflows. ASP.NET Core is also built to be cloud-ready by introducing environment-based configuration and by providing built-in dependency injection support.
+ASP.NET Core is built with the needs of modern Web apps in mind, including a unified story for building Web UI and Web APIs that integrate with today's modern client-side frameworks and development workflows. ASP.NET Core is built to be cloud-ready by introducing environment-based configuration and by providing built-in :doc:`dependency injection </fundamentals/dependency-injection>` support.
```

Comment:
Break the first sentence into two: ASP.NET Core is built with the needs of modern web apps in mind. This includes a unified story, for building both web UIs and web APIs, which integrates modern client-side frameworks and development workflows.